packages/client-web/src/modules/feeds/model/store.ts

```diff
@@ -45,37 +43,6 @@ export const setFeedAuthenticating = (
   value,
 });
 
-export const getOauthUrl = (name: FeedName) => (dispatch: AppDispatch) => {
-  dispatch(setFeedAuthenticating(name, true));
-  return feedsService
-    .getOauthUrl(name)
-    .then((url) => url)
-    .finally(() => dispatch(setFeedAuthenticating(name, false)));
-};
-
-export const authenticateFeed =
-  (name: FeedName, code: string) => (dispatch: AppDispatch) => {
-    dispatch(setFeedAuthenticating(name, true));
-    return feedsService
-      .authenticate(name, { code })
-      .then(() => dispatch(setFeedConnected(name, true)))
-      .finally(() => dispatch(setFeedAuthenticating(name, false)));
-  };
-
-export const disconnectFeed = (name: FeedName) => (dispatch: AppDispatch) => {
-  dispatch(setFeedAuthenticating(name, true));
-  return (
-    feedsService
-      .deauthorize(name)
-      // BE may return a message for further, manual instructions
-      .then((message) => {
-        dispatch(setFeedConnected(name, false));
-        return message;
-      })
-      .finally(() => dispatch(setFeedAuthenticating(name, false)))
-  );
-};
-
```

Comment:
All these thunks have been replaced with plain typescript implementations in the model